### PR TITLE
Include hidden/dot files by default

### PR DESCRIPTION
**Pull request by weshooper**

### CODE REVIEW

File: src/Factory.php [hunk 1]
```diff
@@ -125,8 +127,8 @@ public static function fromDefaults(array $overrideRules = []): Config
             ->in(getcwd())
             ->exclude(static::DEFAULT_EXCLUDED_DIRS)
             ->notName(static::DEFAULT_EXCLUDED_NAME)
-            ->name('*.phtml') // PhpCsFixer adds *.php and *.phpt already
-            ->name('.php_cs.dist'); // @todo: has no effect since Finder ignores hidden files
+            ->name(static::DEFAULT_INCLUDED_NAME)
+            ->ignoreDotFiles(false); // Mainly to avoid missing `.php-cs-fixer.dist.php`
```
Comment (ziadoz) [inline src/Factory.php:131]: Rather than include all dot files, is it worth just adding this specific filename to the list of files we process?

Comment (weshooper) [reply src/Factory.php:131]: That's exactly what this is fixing - we were adding a specific filename, but it had no effect unless all dot files are included. I also think it's highly unlikely we'd have a bunch of hidden `.php` files that we wouldn't want to include. And even if we did, this change doesn't prevent a consumer of this package configuring around.

File: src/Factory.php [hunk 2]
```diff
@@ -11,6 +11,8 @@ class Factory
 
     public const DEFAULT_EXCLUDED_NAME = 'AcceptanceTesterActions.php'; // Annotated with @codingStandardsIgnoreFile
 
+    public const DEFAULT_INCLUDED_NAME = ['*.phtml', '*.phpt', '/\.php$/']; // PhpCsFixer adds *.php already (@todo: remove '/\.php$/' once https://github.com/symfony/symfony/issues/42675 fixed)
```
Comment (liamkeily) [inline src/Factory.php:14]: Is .phpt no longer included by default?

**Review by liamkeily — APPROVED** [review]

**Review by ziadoz — APPROVED** [review]

File: src/Factory.php (hunk 2 above)
Comment (weshooper) [reply src/Factory.php:14]: Correct, was removed a while ago, but seems useful to include.

**Pull request merged**